lib/stencil-download.utils.js

```diff
@@ -0,0 +1,137 @@
+const request = require("request");
+const yauzl = require('yauzl');
+const fs = require('fs');
+const tmp = require('tmp');
+const path = require('path');
+const utils = {};
+
+module.exports = utils;
+
+utils.downloadThemeFiles = (options, callback) => {
```

Comment:
Would love some tests around this until function

Reply:
@junedkazi -  You mean the `stencil-download.utils`. Sounds good!. I will put something in for that.

Reply:
@junedkazi - test has been added!